OrangeDesignSystemDemo/OrangeDesignSystemDemo/Screens/Components/Pages/BottomSheet/Standard/BottomSheetStandardVariant.swift

```diff
@@ -94,13 +90,11 @@ struct RecipesListSelection: View {
 
     var body: some View {
         VStack(spacing: 0) {
-            ForEach(self.listItemModels, id: \.title) { listItemModel in
-                ODSListStandardItem(model: listItemModel)
-                    .padding(.horizontal, ODSSpacing.s)
-                    .listRowSeparator(Visibility.visible)
-                    .listRowInsets(EdgeInsets())
+            ForEach(Self.receipes, id: \.title) { recipe in
```

Comment:
Supposed _title_ will be always unique... not sure of that.

Reply:
Prehaps, to be sure i and a id on recipe struct